R/toolISOhistorical.R

```diff
@@ -152,25 +152,37 @@ toolISOhistorical <- function(m, mapping = NULL, additional_mapping = NULL, over
                              mapping[is.element(mapping$toISO, a$toISO), c("fromISO", "toISO")],
                              weight = weight + 1e-12,
                              negative_weight = "allow")
+        
+        # fill data
+        if (overwrite) {
+          m[a$toISO, subTime, ] <- mTr
+        } else {
+          m[a$toISO, subTime, ][is.na(m[a$toISO, subTime, ])] <- mTr[is.na(m[a$toISO, subTime, ])]
+        }
+        
+        # delete old lines
+        for (b in a$fromISO) {
+          if (is.element(b, getItems(m, dim = 1.1))) {
```

Comment:
I find it a bit easier to read this way, just an option:
```suggestion
          if (b %in% getItems(m, dim = 1.1)) {
```

Reply:
Ah, I did not know that syntax, sounds good!